src/hooks/useLocalStorage.ts

```diff
@@ -11,6 +11,7 @@ function useLocalStorage<V>(
       const item = window.localStorage.getItem(key);
       return item ? JSON.parse(item) : initialValue;
     } catch (error) {
+      // Not shown in UI. Logged for debugging purposses.
```

Comment:
A spelling fix here for all these comments.
```suggestion
      // Not shown in UI. Logged for debugging purposes.
```

Reply:
I've fixed this and also fixed the same issue on line 28. Thanks for the review!